### PR TITLE
chore: proposal to update the terminal output for the login

### PR DESCRIPTION
**Pull request by uriii3**
Trying to followup on [Issue-139](https://github.com/mercator-ocean/copernicus-marine-toolbox/issues/139). Seems like a good idea to point out where the registration is from within the toolbox. 
Not sure if the option of having it in the ReadMe is enough, to be discussed.

### CODE REVIEW

File: copernicusmarine/core_functions/credentials_utils.py
Comment (renaudjester) [inline copernicusmarine/core_functions/credentials_utils.py:1]: Make a little test

File: copernicusmarine/core_functions/credentials_utils.py [hunk 1]
```diff
@@ -91,6 +91,12 @@ def _load_credential_from_motu_configuration_file(
 def _retrieve_credential_from_prompt(
     credential_type: Literal["username", "password"], hide_input: bool
 ) -> str:
+    if credential_type == "username":
+        logger.info(
+            "Downloading CMEMS data requires a Copernicus Marine username "
+            "and password, sign up for free at:"
+            " https://data.marine.copernicus.eu/register."
```
Comment (renaudjester) [inline copernicusmarine/core_functions/credentials_utils.py:98]: without the "."

File: tests/test_command_line_interface.py [hunk 2]
```diff
@@ -831,6 +831,11 @@ def test_login_is_prompt_when_configuration_file_doest_not_exist(
             command, env=environment_without_crendentials
         )
         assert self.output.returncode == 1
+        assert (
+            b"Downloading CMEMS data requires a Copernicus Marine username "
+            b"and password, sign up for free at:"
+            b" https://data.marine.copernicus.eu/register"
+        ) in self.output.stderr
         assert b"username:" in self.output.stdout
```
Comment (renaudjester) [inline tests/test_command_line_interface.py:839]: check that "copernicusmarine username:" is in the stdout
and maybe the password too 🤔

Comment (uriii3) [reply tests/test_command_line_interface.py]: I'm not sure we are entering anything, so not that sure that "coperniscusmarine password" appears there.... but I'll try. Do you prefer that I enter something to check if not?

Comment (renaudjester) [reply tests/test_command_line_interface.py]: Ah yeah true :D You can try by putting an input in the `execute_command`

Comment (uriii3) [reply tests/test_command_line_interface.py]: check the tests and where they failed, it is this! We don't check that the password exists.

**Review by renaudjester — APPROVED** [review]

**Comment by veenstrajelmer** [comment]
Fixes #139 (added this to make sure the PR is linked to the issue)

**Pull request merged**